model/commitqueue/commit_queue.go

```diff
@@ -29,12 +29,13 @@ type CommitQueueItem struct {
 	Issue   string `bson:"issue"`
 	PatchId string `bson:"patch_id,omitempty"`
 	// Version is the ID of the version that is running the patch. It's also used to determine what entries are processing
-	Version             string    `bson:"version,omitempty"`
-	EnqueueTime         time.Time `bson:"enqueue_time"`
-	ProcessingStartTime time.Time `bson:"processing_start_time"`
-	Modules             []Module  `bson:"modules"`
-	MessageOverride     string    `bson:"message_override"`
-	Source              string    `bson:"source"`
+	Version              string    `bson:"version,omitempty"`
+	EnqueueTime          time.Time `bson:"enqueue_time"`
+	ProcessingStartTime  time.Time `bson:"processing_start_time"`
+	Modules              []Module  `bson:"modules"`
+	MessageOverride      string    `bson:"message_override"`
+	Source               string    `bson:"source"`
+	QueueLengthAtEnqueue int       `bson:"queue_length_at_enqueue"`
```

Comment:
Can you add a comment to this field to explain its use, since it's just used for logging?